cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaMirrorMaker2Connectors.java

```diff
@@ -114,6 +115,10 @@ public static KafkaMirrorMaker2Connectors fromCrd(Reconciliation reconciliation,
                     } else if (!existingClusterAliases.contains(mirror.getTargetCluster())) {
                         errorMessages.add("Target cluster alias " + mirror.getTargetCluster() + " is used in a mirror definition, but cluster with this alias does not exist in cluster definitions");
                     }
+
+                    if (!mirror.getTargetCluster().equals(connectCluster)) {
+                        errorMessages.add("Target cluster alias " + mirror.getTargetCluster() + " is used in a mirror definition, but it is not the same as the connect cluster alias " + connectCluster);
```

Comment:
I wonder if users would really understand what it means. I wonder if this would make more sense:

```
Target cluster alias " + mirror.getTargetCluster() + " has to be the same as the Connect cluster alias " + connectCluster
```

WDYT @ppatierno @PaulRMellor?

Reply:
You're right, it could be clearer. Maybe like this?
```
Target cluster alias (currently set to " + mirror.getTargetCluster() + ") has to be the same as the Connect cluster alias " + connectCluster

Reply:
I think that would work. Although I wonder if it would be better to based it the other way => users normally know what their target and source clusters are. They don't realize the target one has to be the connect one. So normally they would I guess change the connect alias. So maybe saying something like this will make it more clear?

```
Connect cluster alias (currently set to " + connectCluster + ") has to be the same as the target cluster alias " + mirror.getTargetCluster()
```

Reply:
Makes sense. I think I parroted the original form from the documentation, but yours is clearer.
> The cluster alias used for Kafka Connect. The value must match the alias of the target Kafka cluster as specified in the spec.clusters configuration.